libretro/libretro-core.c

```diff
@@ -2614,10 +2625,12 @@ size_t retro_serialize_size(void)
             fseek(file, 0L, SEEK_END);
             size = ftell(file);
             fclose(file);
+            unlink(file);

```

Comment:
Please rebase your PR so these erroneous unlinks don’t appear in history, thanks.